### PR TITLE
Update bundle visualization and add export gif to API

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

**Comment by 36000** [comment]
dwi_aligned_trilin_space-RASMM_model-DTI_desc-det-AFQ_viz

**Comment by 36000** [comment]
white_back

**Comment by 36000** [comment]
dwi_aligned_trilin_space-RASMM_model-DTI_desc-det-AFQ_UNC_R_viz

**Comment by 36000** [comment]
dwi_aligned_trilin_space-RASMM_model-DTI_desc-det-AFQ_viz

**Comment by arokem** [comment]
To make the bundle => color mapping more robust, maybe use a dictionary, instead of a list of tuples. Users would have to provide a key as input, so maybe provide another option as well (for example, providing a tuple of RGB values)

**Comment by 36000** [comment]
dwi_aligned_trilin_space-RASMM_model-DTI_desc-det-AFQ_viz
Made using dictionary

File: AFQ/api.py [hunk 1]
```diff
@@ -935,6 +936,88 @@ def _export_bundles(self, row):
...
+                scene = viz.visualize_bundles(
+                    bundles_file,
+                    affine_or_mapping=row['dwi_affine'],
+                    bundle_names=self.bundle_dict,
+                    bundle=uid,
+                    inline=False,
+                    interact=False,
+                    scene=scene)
+            except ValueError:
+                print("No streamlines found for " + bundle_name)
```
Comment (arokem) [inline AFQ/api.py:995]: We'll add logging instead of print in #237

File: requirements.txt [hunk 2]
```diff
@@ -17,3 +17,6 @@ python-dateutil==2.8.0
 toml==0.10.0
 setuptools_scm[toml]==3.4.3
 popylar==0.2
+fury==0.4.0
+imageio==2.8.0
+ipython==7.13.0
```
Comment (arokem) [inline requirements.txt:22]: Need to add a newline at the end of the requirements.txt file.

**Pull request merged**